presto-geospatial/src/main/java/com/facebook/presto/geospatial/GeoFunctions.java

```diff
@@ -113,6 +113,23 @@ public static Slice stAsText(@SqlType(GEOMETRY_TYPE_NAME) Slice input)
         return Slices.utf8Slice(deserialize(input).asText());
     }
 
+    @SqlNullable
+    @Description("Returns a geometry that represents all points whose distance from specified geometry is less than or equal to specified distance")
+    @ScalarFunction("ST_Buffer")
+    @SqlType(GEOMETRY_TYPE_NAME)
+    public static Slice stBuffer(@SqlType(GEOMETRY_TYPE_NAME) Slice input, @SqlType(StandardTypes.DOUBLE) double distance)
```

Comment:
we can static import `StandardTypes.DOUBLE` since we static import the other types in this class